src/inc/MSCOREE.IDL

```diff
@@ -221,7 +224,7 @@ typedef enum
 interface ICLRRuntimeHost : IUnknown
 {
     // Starts the runtime. This is equivalent to CoInitializeCor().
```

Comment:
This is legacy hosting interface that number of folks out there still depends on. It is even mentioned in the docs: https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting

 Avoid changing it